share/completions/rc-status.fish

```diff
@@ -0,0 +1,17 @@
+complete -f rc-status
+complete -c rc-status -n "test (__fish_number_of_cmd_args_wo_opts) = 1" \
+    -xa "(/bin/ls -1 /etc/runlevels)"
```

Comment:
I don't love that we use `ls` here, or that we hard-code it as being in /bin.

Unfortunately the current builtin-only way is:

```fish
set -l levels /etc/runlevels/*
string replace /etc/runlevels/ $levels
```

with #8265 this would become:

```fish
path basename /etc/runlevels/*
```

Anyway, for now the ls is probably fine as it is.